docker-compose.yaml

```diff
@@ -6,6 +6,8 @@ services:
       dockerfile: docker/Dockerfile
     volumes:
     - ./:/usr/src/myapp
+    depends_on:
+      - otel-collector
```

Comment:
I recognize this might be a slight bit of a bikeshedding discussion, but should our default depend on the collector?  I almost think it would make more sense to have a separate docker-compose file for something that depends on the connector.  I'm not strongly opinionated on this.

Reply:
Happy to change this.  I initially thought to use this `docker-compose` because even though the collector isn't configured for it now, it could be updated to send to the zipkin and jaeger services as well, but there is a good argument for not having the php service be dependent for other cases.  I created the new yaml which also diversifies our examples so overall a good thing :)